packages/ndla-ui/src/locale/messages-en.ts

```diff
@@ -1192,18 +1192,30 @@ const messages = {
         title: 'Categories',
         posts: 'Posts',
       },
+      new: {
+        post: 'Write a reply',
+        topic: 'New topic',
+      },
+      warning: {
+        post: 'Your answer is visible for every teacher who is logged on to Min NDLA.',
+        topic: 'Your topic is visible for every teacher who is logged on to Min NDLA.',
```

Comment:
To eller for? Blei litt usikker.

Reply:
Synes `for` høres mer riktig ut

Reply:
Leita litt, fant denne setninga som virker å oppsummere det greit:

> Visible to is a statement of the bald fact that something can be seen and visible for is a statement that the visibility is intentional.

Så da ville jeg sagt at `for` er riktig.